accessible/appview.component.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Blockly Demos: AccessibleBlockly
+ *
+ * Copyright 2016 Google Inc.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Angular2 Component that details how the AccessibleBlockly app is rendered on the page.
+ * @author madeeha@google.com (Madeeha Ghori)
+ */
+var blocklyApp = blocklyApp || {};
+blocklyApp.workspace = blocklyApp.workspace || new Blockly.Workspace();
+
+blocklyApp.AppView = ng.core
+  .Component({
+    selector: 'blockly-app',
+    template: `
+    <table>
+    <tr>
+      <td class='blocklyTable'>
+        <toolbox-view>Loading Toolbox...</toolbox-view>
+      </td>
+      <td class='blocklyTable'>
+        <workspace-view>Loading Workspace...</workspace-view>
+      </td>
+    </tr>
+    </table>
+    <label id='blockly-block-summary' aria-hidden='true' hidden>block summary</label>
```

Comment:
Elsewhere, in HTML files in the blockly repo, we use double quotation marks. It would be nice to do that here and elsewhere for consistency.